test/plugins/test_lambda_release.py

```diff
@@ -0,0 +1,128 @@
+import unittest
+
+from datetime import datetime
+from mock import Mock, patch
+from cdflow_commands.plugins.aws_lambda import Release
+
+
+class TestLambdaRelease(unittest.TestCase):
+
+    @patch('cdflow_commands.plugins.aws_lambda.os')
+    @patch('cdflow_commands.plugins.aws_lambda.ZipFile')
+    def test_release_create_zips_directory(self, zip_file, mock_os):
+        config = Mock()
```

Comment:
Some of the places we use `Mock` I think we should probably use `MagicMock` (https://docs.python.org/3/library/unittest.mock.html#magic-mock) and pass a `spec` for the object we're mocking e.g.
```python
config = MagicMock(spec=ReleaseConfig)
```